<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Whats Up Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSent">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar Mensaje</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnArchivo">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScan">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Escanear red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInterfaces">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar en:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbInterfaces">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblChat">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat usando la interfaz: #</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDisponibles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Host e IP's disponibles</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblChatWith">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Conversando con: #</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtMessage">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>371</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Escribe:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
